.github/workflows/build.yml

```
@@ -32,6 +32,6 @@ jobs:
         # MacOS arm64 wheels
         - cp3*-macosx_arm64
       sdist: true
-      test_command: python -c "from jwst.lib import winclip; from jwst.cube_build import cube_match_internal, cube_match_sky_pointcloud, cube_match_sky_driz, blot_median; from jwst.straylight import calc_xart"
+      test_command: python -c "${{((github.event_name == 'release') && (github.event.action == 'released')) && 'from jwst.lib import winclip; from jwst.cube_build import cube_match_internal, cube_match_sky_pointcloud, cube_match_sky_driz, blot_median; from jwst.straylight import calc_xart; from jwst import crds_context' || 'from jwst.lib import winclip; from jwst.cube_build import cube_match_internal, cube_match_sky_pointcloud, cube_match_sky_driz, blot_median; from jwst.straylight import calc_xart' }}"
```

I think this would be better:
```suggestion
      test_command: python -c "${{ format('{0}{1}', (github.event_name == 'release' && github.event.action == 'released') && 'from jwst import crds_context; ' || '', 'from jwst.lib import winclip; from jwst.cube_build import cube_match_internal, cube_match_sky_pointcloud, cube_match_sky_driz, blot_median; from jwst.straylight import calc_xart') }}"
```